app/controllers/json_api/people_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2022, Schweizer Wanderwege. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class JsonApi::PeopleController < JsonApiController
+
+  def index
+    authorize!(:index_people, Group)
+    people = PersonResource.all(params, index_people_scope)
+    render(jsonapi: people)
+  end
+
+  def show
+    authorize!(:show, entry)
+    person = PersonResource.find(params)
+    render(jsonapi: person)
+  end
+
+  def update
+    authorize!(:update, entry)
+    resource = PersonResource.find(params)
```

Comment:
Wäre es nicht sicherer so? Wegen den Fähigkeiten von Graphiti bin ich nicht zu 100% sicher dass `entry` und `resource.data` immer dasselbe Model repräsentieren.
```suggestion
    resource = PersonResource.find(params)
    authorize!(:update, resource.data)
```

Reply:
yep, das ist wohl so. 
wir validieren das aber jetzt via die Resource find methode. 
so stellen wir sicher das param id immer der data attributes id entspricht.